Documentation/deployment-and-configuration.md

```diff
@@ -2,15 +2,51 @@
 
 Deploying `fleet` is as simple as dropping the `fleetd` binary on a machine with access to etcd and starting it.
 
-Deploying `fleet` on CoreOS is even simpler: just run `systemctl start fleet`. The built-in configuration assumes each of your hosts is serving an etcd endpoint at one of the default locations (http://127.0.0.1:4001 or http://127.0.0.1:2379). However, if your etcd cluster differs, you must make the corresponding configuration changes.
+Deploying `fleet` on CoreOS is even simpler: just run `systemctl start fleet`. The built-in configuration assumes each of your hosts is serving an etcd endpoint at one of the default locations (http://127.0.0.1:2379 or http://127.0.0.1:4001). However, if your etcd cluster differs, you must make the corresponding configuration changes.
 
 ## etcd
 
-Each `fleetd` daemon must be configured to talk to the same [etcd cluster][etcd]. By default, the `fleetd` daemon will connect to either http://127.0.0.1:4001 or http://127.0.0.1:2379, depending on which endpoint responds. Refer to the configuration documentation below for customization help.
+Each `fleetd` daemon must be configured to talk to the same [etcd cluster][etcd]. By default, the `fleetd` daemon will connect to either http://127.0.0.1:2379 or http://127.0.0.1:4001, depending on which endpoint responds. Refer to the configuration documentation below for customization help.
 
-`fleet` requires etcd be of version 0.3.0+.
+`fleet` requires etcd be of version 0.3.0+ but it is recommended to use etcd 2.0.0+ which supports [TLS authentication][etcd-security].
 
-[etcd]: https://coreos.com/docs/cluster-management/setup/getting-started-with-etcd
+### TLS Authentication
+
+If you have etcd cluster with enabled TLS authentication you have to configure fleet to use TLS keypair. This example will should you how to do that:
```

Comment:
If your etcd cluster has TLS authentication enabled, you will need to configure fleet to use an appropriate TLS keypair. The example shows below how to achieve this: